bokehjs/src/coffee/models/mappers/log_mapper.coffee

```diff
@@ -20,11 +21,14 @@ export class LogMapper extends Model
     if inter_scale == 0
       value = 0
     else
-      value = (Math.log(x) - inter_offset) / inter_scale
-      if isNaN(value) or not isFinite(value)
-        value = 0
+      clamped_x = clamp(x, @source_range.start, @source_range.end)
```

Comment:
Looking at the tests, it looks like we were only left-side clamping. I'd assume it was a hack to handle negative values, which has been obviated with this PR. I think I should remove the clamping.